lib/modules.nix

```diff
@@ -407,13 +413,16 @@ rec {
       };
     defsFinal = defsFinal'.values;
 
+    invalidDefinitionExceptionOrNull =
```

Comment:
@roberth I don't think this needs to be exposed though, it can be a `let in` within the `mergedValue` instead

Reply:
Didn't realize it was exposed. Yes, that is ok.

Reply:
Found an imo better way to do this instead, which I'm now using for both changes here